sssom/parsers.py

```diff
@@ -499,6 +518,144 @@ def from_obographs(
+    of any limitations of the map record and c) additional classification guidance for the coding professional.,
+    - mapTarget,String,The target ICD-10 classification code of the map record.,
+    - correlationId,SctId,A child of |Map correlation value| in the metadata hierarchy, identifying the correlation
+    between the SNOMED CT concept and the target code.,
+    - mapCategoryId,SctId,Identifies the SNOMED CT concept in the metadata hierarchy which is the MapCategory for the
+    associated map record. This is a subtype of 447634004 |ICD-10 Map Category value|.,
+    """
+    # https://www.findacode.com/snomed/447561005--snomed-ct-source-code-to-target-map-correlation-not-specified.html
+    match_type_snomed_unspecified_id = 447561005
+    prefix_map = _ensure_prefix_map(prefix_map)
+    ms = _init_mapping_set(meta)
+
+    mlist: List[Mapping] = []
+    for _, row in df.iterrows():
+        mdict = {
+            'subject_id': f'SNOMED:{row["referencedComponentId"]}',
+            'subject_label': row['referencedComponentName'],
+
+            # 'predicate_id': 'skos:exactMatch',
+            # - mapCategoryId: can use for mapping predicate? Or is correlationId more suitable?
+            #   or is there a SKOS predicate I can map to in case where predicate is unknown? I think most of these
+            #   mappings are attempts at exact matches, but I can't be sure (at least not without using these fields
+            #   to determine: mapGroup, mapPriority, mapRule, mapAdvice).
+            # mapCategoryId,mapCategoryName: Only these in set: 447637006 "MAP SOURCE CONCEPT IS PROPERLY CLASSIFIED",
+            #   447638001 "MAP SOURCE CONCEPT CANNOT BE CLASSIFIED WITH AVAILABLE DATA",
+            #   447639009 "MAP OF SOURCE CONCEPT IS CONTEXT DEPENDENT"
+            # 'predicate_modifier': '???',
+            #   Description: Modifier for negating the prediate. See https://github.com/mapping-commons/sssom/issues/40
+            #   Range: PredicateModifierEnum: (joe: only lists 'Not' as an option)
+            #   Example: Not Negates the predicate, see documentation of predicate_modifier_enum
+            # - predicate_id <- mapAdvice?
+            # - predicate_modifier <- mapAdvice?
+            #   mapAdvice: Pipe-delimited qualifiers. Ex:
+            #   "ALWAYS Q71.30 | CONSIDER LATERALITY SPECIFICATION"
+            #   "IF LISSENCEPHALY TYPE 3 FAMILIAL FETAL AKINESIA SEQUENCE SYNDROME CHOOSE Q04.3 | MAP OF SOURCE CONCEPT
+            #   IS CONTEXT DEPENDENT"
+            #   "MAP SOURCE CONCEPT CANNOT BE CLASSIFIED WITH AVAILABLE DATA"
+            'predicate_id': f'SNOMED:{row["mapCategoryId"]}',
+            'predicate_label': row['mapCategoryName'],
+
+            'object_id': f'ICD10CM:{row["mapTarget"]}',
+            'object_label': row['mapTargetName'],
+
+            # match_type <- mapRule?
+            #   ex: TRUE: when "ALWAYS <code>" is in pipe-delimited list in mapAdvice, this always shows TRUE. Does this
```

Comment:
### `match_type` or `predicate_id` from `mapRule`
@matentzn This in particular I might want to go back and change. The `mapRule` column is a `bool`. I _think_ it may be the case that all instances of `TRUE` in this column could indicate that they are exact matches, because this field only shows `TRUE` when the `mapAdvice` field includes text that says `ALWAYS <code>`.

But by "ALWAYS" and "TRUE", I don't know if that means that (a) the mapped SNOMED::ICD code are the only codes that are related to each other, or (b) it's an exact match, in which case we could use `skos:exactMatch` here for that (also, does SKOS not have integer IDs for its concepts?).

I'm thinking that (b) is more likely the case, but I'm not too sure. Maybe someone with more domain expertise could look at some of the example cases. As mentioned [here](https://www.nlm.nih.gov/research/umls/mapping_projects/snomedct_to_icd10cm.html), the map TSV can be downloaded here:
> The map is also available in a human readable format (tls_Icd10cmHumanReadableMap_US1000124_YYYYMMDD.tsv) and can be downloaded from the [US Edition page](https://www.nlm.nih.gov/healthit/snomedct/us_edition.html).

Reply:
Let's do this in our next 1:1 rather than discuss it in this PR. You are asking all the right questions, but answers are complicated and its better done in person.

Reply:
Sounds good. added to my agenda notes

Reply:
Sounds good. added my agenda